lib/inject.js

```diff
@@ -3,6 +3,13 @@
 /* eslint-disable max-statements */
 function injectPa11y(window, options, done) {
 
+	if (options.verifyPage) {
+		var windowHtml = window.document.documentElement.outerHTML;
+		if (!windowHtml.match(new RegExp(options.verifyPage))) {
+			return reportError('Page not verified');
```

Comment:
I'd maybe make this more descriptive. Something like `The page can not be verified, it was expected to match the following regular expression: <regexp>`?